Домашка №3

aleroma opened the pull request:
По лэйауту - три части PageHeader, MainContent и PageFooter.
В MainContent - RestaurantButtons и RestaurantInfo.
Состояние активного ресторана activeId хранится в компоненте MainContent
и передается соответственно в RestaurantButtons и RestaurantInfo.
RestaurantButtons подсвечивает активную кнопку.
RestaurantInfo фильтрует и выводит один ресторан соответствующий activeId.
Счетчик - согласно ДЗ.

Commit:
Добавил папки и файлы компонентов App и Layout

Commit:
pre hw-3

Commit:
работающие табы

Commit:
hw-3

zubkov7 commented on src/Restaurant/DishCounter.jsx:
```diff
@@ -0,0 +1,23 @@
+
+export const DishCounter = () => {
+    const [counter, setCounter] = useState(0);
+    const minCount = 0;
+    const maxCount = 5;
```
нужно вынести из компонента, иначе будут пересоздавать на каждый ререндер

Commit:
fix-1

Вынес minCount и maxCount за пределы компонента чтобы не пересоздавались при каждом рендере.

aleroma commented:
@zubkov7 Вынес minCount и maxCount за пределы компонента чтобы не пересоздавались при каждом рендере.
